src/components/TopicMenu/TopicMenu.js

```diff
@@ -207,12 +210,14 @@ class TopicMenu extends PureComponent {
               </div>
               {t(topic.name)}
             </div>
-            <div
-              className={`wkp-layer-toggler ${collapsed ? 'collapsed' : ''}`}
-              style={{
-                display: topic.key === activeTopic.key ? 'block' : 'none',
-              }}
-            />
+            {menuVisibleLayers.length ? (
```

Comment:
I discover this works:
```suggestion
            {!!menuVisibleLayers.length && (
```